tools/cloudbuild-artifacts.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- name: golang:1.14-alpine
+  args: ['go', 'build', './...']
+- name: golang:1.14-alpine
+  args: ['go', 'test', './...']
+- name: alpine
+  args: ['./tools/package.sh', '$COMMIT_SHA']
+options:
+  env:
+  - CGO_ENABLED=0
```

Comment:
Is this necessary?

Reply:
Without it the binary will depend on libc (for things like dns). The binary produced here is what we ship to users. Avoiding libc means we are able to use Docker's sketch (which is empty) instead of needing an actual distro with libc.